Add RISW slides and new sticker

danielinteractive opened the pull request.

kkmann commented:
Love it, much better hex + name now as well ;)

Curious about the feedback on that presenation!

ywang-gilead commented on slides/risw-first-year-wg-sep2023.qmd:
```diff
@@ -0,0 +1,229 @@
+- MMRM is a popular analysis method for longitudinal continuous outcomes in randomized clinical trials
+- No tailored R package with sufficient capabilities/reliability
+- Also used as backbone for more recent methods such as multiple imputation
+- We have run comparison analyses with other R packages, namely `nlme`, `glmmTMB` and `lme4` as well as SAS `PROC GLIMMIX` and have found that `mmrm` is the fastest of them and provides closes results to `PROC GLIMMIX`
```
Our current vignette compares with SAS `PROC MIXED`, did we also run comparison with `PROC GLIMMIX`?

danielinteractive replied:
Yeah we heard from @yonicd that this one is newer. Ongoing PR in mmrm updates the comparison to using this

ywang-gilead replied:
Thanks Daniel.

wiligl commented:
- Great slides, many aspects covered 
- Cool new name, logo can be improved :-)
- Validation, documentation, user-friendliness, training are all needed to produce correct results, which the user intended. Therefore, I support to go beyond validation, emphasizing documentation, but the other aspects could also be added.
Validation and documentation are not sufficient because if a process is correct and fully documented, but complex, confusing, or documentation not user-friendly (eg alphabetic order instead of logic order (cf package manual vs vignette), very long document, not searchable) users may not read, may not find, or misunderstand the information, leading to errors. Therefore, user-friendliness (of software and documentation) and training are an important design features.
-I suggest to support the slide deck in the future (will take some time to get the numbers) with economic data to quantify the business impact of introducing R in a pharma company. The "friendly elves" narrative ("contribute and you will get back more in the future") is likely to motivate statisticians and to build a community, but will not convince business leaders.

danielinteractive commented:
Thanks @wiligl for having a look!

If you like you can propose a new slide or a modification of existing slides re: validation reference I made.

For the logo please get in touch with @ellessenne if you have ideas. (I like it but it is a matter of taste)

jmanitz commented:
Thanks for putting the slides together, looks very nice. 

Some minor comments regarding the RvH: 
slide 6: Is it worth mentioning the R validation Hub? https://www.pharmar.org/
slide 8/9: IMHO the RvH is more about defining a process for risk assessment of R packages rather than specifically defining good practices. Basically answering the question: HOW can I differentiate between a reliable and less reliable package not WHAT differentiates those packages. 
Mission: "R Validation Hub is a collaboration to support the adoption of R within a biopharmaceutical regulatory setting"
Developes tools that supports risk assessment of R packages

danielinteractive commented:
Thanks @jmanitz for your input, much appreciated!
I modified the slides accordingly.
Re: slide 8/9 - I modified the description of RvH. Note that most of these 2 slides is more generally about validation (e.g. I took the definition from https://www.pharmar.org/overview/) and how we relate to this topic (and to me slide 9 makes a lot of sense, what do you think?), RvH is only a pointer to this important initiative.

ellessenne commented:
@wiligl feel free to send me an e-mail with suggestion regarding the logo – happy to tweak and adjust as needed!
